datadog/resource_datadog_dashboard_log_stream_test.go

```diff
@@ -120,3 +121,71 @@ func TestAccDatadogDashboardLogStream(t *testing.T) {
 func TestAccDatadogDashboardLogStream_import(t *testing.T) {
 	testAccDatadogDashboardWidgetUtil_import(t, datadogDashboardLogStreamConfig, "datadog_dashboard.log_stream_dashboard")
 }
+
+const datadogDashboardLogStreamLogSetConfig = `
+resource "datadog_dashboard" "log_stream_dashboard_logset" {
+	title         = "Acceptance Test Log Stream Widget Dashboard"
+	description   = "Created using the Datadog provider in Terraform"
+	layout_type   = "free"
+	is_read_only  = "true"
+
+	widget {
+		log_stream_definition {
+			title = "Log Stream"
+			title_align = "right"
+			title_size = "16"
+			show_message_column = "true"
+			message_display = "expanded-md"
+			query = "status:error env:prod"
+			show_date_column = "true"
+			logset = "main"
```

Comment:
I believe logset should be a string representation of a numeric id, like - https://github.com/terraform-providers/terraform-provider-datadog/blob/master/datadog/resource_datadog_dashboard_test.go#L872